### ISSUE TITLE
Screen reader doesn't announce the tooltip information when focus moves to "Add a trigger" button.

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#7282]
### Describe the issue

repro step:
1. Create a new workflow
2. Observe and verify Screen reader doesn't announce the tooltip information when focus moves to "Add a trigger" button

Actual Result:
 Screen reader doesn't announce the tooltip information when focus moves to "Add a trigger" button.

Observation:  
When screen reader(NVDA,Narrator)  focus lands on the "Add a trigger" control and it just announcing the "Add a trigger" button but not announcing the tooltip information.
JAWS announcing the tooltip information.
Expected Result:
 Screen reader should announce the tooltip information when focus moves to "Add a trigger" button.

![Image](https://github.com/user-attachments/assets/2e0dd245-757b-4d90-a10d-28b31f3c6736)

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

edge
